test/test_evaluate/test_pyc.py

```diff
@@ -45,20 +45,5 @@ def generate_pyc():
             shutil.copy(os.path.join("dummy_package/__pycache__", f), dst)
 
 
-# Python 2.6 does not necessarily come with `compileall.compile_file`.
-@pytest.mark.skipif("sys.version_info > (2,6)")
-@cwd_at('test/test_evaluate')
-def test_pyc():
-    """
-    The list of completion must be greater than 2.
-    """
-    try:
-        generate_pyc()
-        s = jedi.Script("from dummy_package import dummy; dummy.", path='blub.py')
-        assert len(s.completions()) >= 2
-    finally:
-        shutil.rmtree("dummy_package")
-
-
```

Comment:
Can you not remove this test? Sorry I think the check is the wrong way around 
and this was already fixed in the virtualenv branch (will only result in merge
conflicts).